### DISCUSSION TITLE
zitadel working fine

### DISCUSSION THREAD

**Discussion by adriy-be** [9p4/jellyfin-plugin-sso#168] [General]
Hi,
I have tested with zitadel, and it's working well !
Just a remark, the branded part was hidden by brave shield, and probably other tool like ublock.

I don't have created an issue due to is not an issue.

Thanks

**Comment by adriy-be** [comment]
Just a comment, all my configuration pass actually by https. Have a custom url for auth and token is a must-have for internal docker network comunication.
Follow https://github.com/go-gitea/gitea/issues/29317

So, I'm new in the world of SSO/OpenID, it's possible that a bad practice?

**Reply by 9p4** [reply]
Are you using Nginx?

**Reply by adriy-be** [reply]
No, I'm using Caddy

**Comment by samumatic** [comment]
Hey, i have some problems implementing zitadel with the plugin.
I chose the Web(OIDC) -> Code type for it.
After setting everything up, i set the redirect settings to: "https://jelly.lab.domain.de" - my jellyfin URL.
But after clicking the button for "https://jelly.lab.domain.de/sso/OID/start/zitadel" the JSON answer states that:

> The requested redirect_uri is missing in the client configuration. If you have any questions, you may contact the administrator of the application.

But in the URL that part seems ok: `redirect_uri=https%3A%2F%2Fjelly.lab.domain.de%2Fsso%2FOID%2Fredirect%2Fzitadel`

How did you set it up? Did i made a mistake here?

**Reply by samumatic** [reply]
Okay, resolved by using the exact path: `https://jelly.lab.domain.de/sso/OID/redirect/zitadel`.
